v2/internal/util/postgresql/psqluser.go

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package psqluser
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"github.com/Azure/azure-service-operator/v2/internal/set"
+	"strings"
+
+	_ "github.com/lib/pq" //the pg lib
```

Comment:
Wondering if we should consider using https://github.com/jackc/pgx rather than github.com/lib/pq - given that pq is in maintenance mode?

Reply:
I switched the driver to pgx and tested it with success.